pkg/cmd/env/options.go

```diff
@@ -123,6 +128,24 @@ func (o *options) Run(f util.Factory) error {
 		}
 
 		return o.runKubernetes(f.Context(), manager)
+
+	case "loki":
+		t := o.CurrentTarget.WithControlPlane(true)
+		ctx := f.Context()
+
+		if !t.ControlPlane() {
+			return target.ErrNoControlPlaneTargeted
+		}
+
+		client, err := manager.SeedClient(ctx, t)
```

Comment:
```
(⎈ |garden--gcp-external:default)❯ eval $(gardenctl loki-env zsh)
Error: failed to create seed cluster client: no Auth Provider found for name "gcp"
```
this happened when targeting a managed seed and it's seed was a soil cluster that was running on gke